templates/base.html.twig

```diff
@@ -4,7 +4,7 @@
 <!--[if lte IE 7]><html class="no-js ie7 ie67 ie678"{% if lang is not empty %} lang="{{ lang }}"{% endif %}><![endif]-->
 <!--[if IE 8]><html class="no-js ie8 ie678"{% if lang is not empty %} lang="{{ lang }}"{% endif %}><![endif]-->
 <!--[if gt IE 8]><html class="no-js"{% if lang is not empty %} lang="{{ lang }}"{% endif %}><![endif]-->
-<html{% if lang is not empty %} lang="{{ lang }}"{% endif %} class="{{ theme_class() }}" data-controller="dark-theme">
+<html{% if lang is not empty %} lang="{{ lang }}"{% endif %} class="{{ theme_class() }}" data-controller="dark-theme shortcuts">
```

Comment:
decided to have this controller registered at the root level because Mousetrap has no clean way to destroy itself.
for now its has no impact as we still full page reload, but when I'll introduce Turbo soon after this, it means the Mousetrap instance will be kept across navigate.
that's why I prepared already the code for that by mapping all shortcuts at the root level, and make then do something or not if possible, like if the target link or button is on the page